tests/api/test_pushnotification_api.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import annotations
+
+from datetime import datetime
+
+import pytest
+from dateutil.relativedelta import relativedelta
+from django.test.client import Client
+from django.urls import reverse
+
+from integreat_cms.cms.models import (
+    Language,
+    PushNotification,
+    PushNotificationTranslation,
+    Region,
+)
+
+REGION_SLUG = "artland"
+
+
+def create_a_pushnotification(region_slug: str) -> int:
+    """
+    A function to create a new push notification and a trasnaltion in the defaut language of the region.
+    """
+    region = Region.objects.get(slug=region_slug)
+    default_language = Language.objects.get(slug=region.default_language.slug)
+    pushnotification = PushNotification.objects.create(
+        channel="news",
+        draft=False,
+        sent_date=datetime.now() - relativedelta(days=1),
+        created_date=datetime.now() - relativedelta(days=1),
+        scheduled_send_date=None,
+        is_template=False,
+        template_name=None,
+    )
+    pushnotification.regions.add(region)
+    pushnotification.save()
+    pushnotification_translation = PushNotificationTranslation.objects.create(
+        title="German Traslation",
+        text="Hell World",
+        language=default_language,
+        push_notification=pushnotification,
+        last_updated=datetime.now() - relativedelta(days=1),
+    )
+    pushnotification_translation.save()
+
+    return pushnotification.id
+
+
+@pytest.mark.django_db
+def test_no_archived_pushnotification(load_test_data: None) -> None:
+    """
+    Check that archived push notifications are excluded from the API result
+
+    :param load_test_data: The fixture providing the test data (see :meth:`~tests.conftest.load_test_data`)
+    """
+
+    client = Client()
+
+    kwargs = {"region_slug": REGION_SLUG, "language_slug": "de"}
+    api = reverse("api:sent_push_notifications", kwargs=kwargs)
+    start_response = client.get(api)
+    start_result = start_response.content.decode("utf-8")
+
+    pushnotification_id = create_a_pushnotification(REGION_SLUG)
+
+    response_after_new_pn = client.get(api)
+    result_after_new_pn = response_after_new_pn.content.decode("utf-8")
+    assert (
+        '"id": 11, "title": "German Traslation", "message": "Hell World"'
```

Comment:
```suggestion
        '"id": 11, "title": "German Traslation", "message": "Hello World"'
```
If it was intended to have "Hell World" you can ignore this comment :)